src/nvlink/wrapper_32bit.py

```
@@ -0,0 +1,583 @@
+        self._nvlink = None
+        self._is_open = False
+
+        try:
+            import win32com.client
+            import win32com.client.dynamic
+
+            # 32-bit Python: Direct COM object creation (no DllSurrogate needed)
+            # Use dynamic dispatch (late binding) to avoid gen_py cache issues
+            self._nvlink = win32com.client.dynamic.Dispatch("NVDTLabLib.NVLink")
+            logger.info("NV-Link COM object created (32-bit direct)", sid=sid)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)